websites/V/VK/dist/metadata.json

```diff
@@ -7,6 +7,10 @@
     {
       "name": "Bas950",
       "id": "241278257335500811"
+    },
+    {
+      "name": "MrZillaGold",
+      "id": "342146975065702400"
```

Comment:
You can't add ourself as author/contributor since you didn't do anything. (Owner is only one)